packages/utilities/src/scroll.ts

```diff
@@ -66,24 +74,12 @@ const _makeElementScrollAllower = () => {
     }
   };
 
-  return (element: HTMLElement | null, events: EventGroup): void => {
-    if (!element) {
-      return;
-    }
-
-    events.on(element, 'touchstart', _saveClientY, { passive: false });
-    events.on(element, 'touchmove', _preventOverscrolling, { passive: false });
+  events.on(element, 'touchstart', _saveClientY, { passive: false });
+  events.on(element, 'touchmove', _preventOverscrolling, { passive: false });
```

Comment:
I noticed the callers do not invoke `EventGroup.dispose` on `componentWillUnmount`, should they to prevent additional leaks? cc: @dzearing 

https://github.com/OfficeDev/office-ui-fabric-react/blob/8111845351329a689c13050dd7c881089bbea140/packages/office-ui-fabric-react/src/components/Panel/Panel.base.tsx

https://github.com/OfficeDev/office-ui-fabric-react/blob/b37e5730260552b5f4ef6e1a6d6d6e289792830f/packages/office-ui-fabric-react/src/components/Modal/Modal.base.tsx

Reply:
`BaseComponent` calls dispose on willunmount.

Reply:
You're right. Let's keep in mind this area when `BaseComponent` is removed which my understanding is planned. During review of that refactor we must make sure event handlers are properly disposed. Thanks for the fix!